nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -845,6 +845,10 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The `vscode-extensions.ionide.ionide-fsharp` package has been updated to 6.0.0 and now requires .NET 6.0.
 
+- The `libdwarf_0_4` package has been added:
+  - Including `libdwarf` and `dwarfdump` 0.4.0
+  - Supporting all unix platforms
```

Comment:
I don't think that is worth an announcement. People not explicitly developing for libdwarf do not care about this and it is also not a breaking change.

Reply:
Announcement removed